pyparsing/core.py

```diff
@@ -2829,78 +2837,81 @@ def __init__(
             self.errmsg += " as a keyword"
 
         # see if we can make a regex for this Word
-        if " " not in (self.initChars | self.bodyChars):
```

Comment:
I think these are all good, but would like to see as a separate PR to test in isolation.